tests/index_test.js

```diff
@@ -204,6 +204,8 @@ describe('StyleSheet.create', () => {
     it('works for empty stylesheets and styles', () => {
         const emptySheet = StyleSheet.create({});
 
+        assert.ok(emptySheet);
```

Comment:
I think this is reasonable. I guess we're really testing that `StyleSheet.create({})` isn't throwing or something, but this seems like a reasonable way to say "make sure emptySheet exists at least".